net/modemmanager/files/modemmanager.proto

```diff
@@ -364,6 +370,50 @@ modemmanager_set_allowed_mode() {
 	}
 }
 
+modemmanager_check_state() {
+	local device="$1"
+	local modemstatus="$2"
+	local pincode="$3"
+
+	local state reason
+
+	state="$(modemmanager_get_field "${modemstatus}" "state")"
+	state="${state%% *}"
+	reason="$(modemmanager_get_field "${modemstatus}" "state-failed-reason")"
+
+	case "$state" in
+		"failed")
+			case "$reason" in
+				"sim-missing")
+					echo "SIM missing"
+					proto_notify_error "${interface}" MM_FAILD_REASON_SIM_MISSING
```

Comment:
Typo `MM_FAILD` -> `MM_FAILED`

Reply:
Nice catch. Will update on the next `git push --force`.